lua/crates/toml.lua

```diff
@@ -774,10 +777,17 @@ function M.parse_crates(buf)
             dep_section = M.parse_section(section_text, line_nr, section_start - 1)
             dep_section_crate = nil
             if dep_section then
+                if dep_section.name then
```

Comment:
It would be awesome crates defined as tables would be completed too, but currently this happens:
```toml
[dependencies.name]
#                 ^ cursor here
```
the version get's inserted after the name
```toml
[dependencies.name = "0.1.0"]
```
instead of below:
```toml
[dependencies.name]
version  = "0.1.0"
```
I would also be fine with disabling the completion for now and adding this back in a follow up PR though :)